Resources/Prototypes/Loadouts/Jobs/Science/borg.yml

```diff
@@ -0,0 +1,29 @@
+- type: loadout
+  id: BorgGeneric
+  entity: PlayerBorgGeneric
+  entityDummy: DummyBorgGeneric
+
+- type: loadout
+  id: BorgMining
+  entity: PlayerBorgMining
+  entityDummy: DummyBorgMining
+
+- type: loadout
+  id: BorgEngineer
+  entity: PlayerBorgEngineer
+  entityDummy: DummyBorgEngineer
+
+- type: loadout
+  id: BorgJanitor
+  entity: PlayerBorgJanitor
+  entityDummy: DummyBorgJanitor
+
+- type: loadout
+  id: BorgMedical
+  entity: PlayerBorgMedical
+  entityDummy: DummyBorgMedical
+
+- type: loadout
+  id: BorgService
```

Comment:
Is there any reason you need the dummy prototypes?

Reply:
The loadout name is the dummy name.
Every species uses it so it doesn't have to load every system for performance. Just applied the same idea to the loadout.